src/pages/research/index.tsx

```diff
@@ -1,14 +1,453 @@
+          referenceHref="https://doi.org/10.1145/3545945.3569847"
+        />
+
+        <ResearchCard
+          title="A generalized framework for describing question randomization"
+          reference="R. Mahinpei, I. Xu, S. Wolfman, F. Moosvi. SIGCSE 2024"
+          referenceHref="https://doi.org/10.1145/3626253.3635599"
+        />
+      </React.Fragment>
+    ),
+  },
+  {
+    title: "Retrieval practice and second-chance testing",
+    contents: (
+      <React.Fragment>
+        <ResearchCard
+          title="Frequent mastery testing with second-chance exams leads to enhanced student learning in undergraduate STEM"
+          reference="J. Morphew, M. Silva, G. Herman, and M. West. Applied Cognitive Psychology 2019."
+          referenceHref="https://doi.org/10.1002/acp.3605"
+        />
+
+        <ResearchCard
+          title="Second-chance testing course policies and student behavior"
+          reference="G. Herman, K. Varghese, and C. Zilles. FIE 2019."
+          referenceHref="https://ieeexplore.ieee.org/document/9028490"
+        />
+
+        <ResearchCard
+          title="Comparison of grade replacement and weighted averages for second-chance exams"
+          reference="G. Herman, Z. Cai, T. Bretl, C. Zilles, and M. West. ICER 2020."
+          referenceHref="https://dl.acm.org/doi/10.1145/3372782.3406260"
+        />
+
+        <ResearchCard
+          title="Students perceptions and behavior related to second-chance testing"
+          reference="C. Emeka, T. Bretl, G. Herman, M. West, and C. Zilles. FIE 2021."
+          referenceHref="https://ieeexplore.ieee.org/document/9637173"
+        />
+
+        <ResearchCard
+          title="Determining the best policies for second-chance tests for STEM students"
+          referenceHref="https://peer.asee.org/43019"
+          reference="C. Emeka, D. Smith, C. Zilles, M. West, G. L. Herman, and T. Bretl. ASEE 2023."
+        />
+
+        <ResearchCard
+          title="Second-chance testing as a means of reducing students' test anxiety and improving outcomes"
+          referenceHref="https://peer.asee.org/44207"
+          reference="C. Emeka, C. Zilles, M. West, G. Herman, and T. Bretl. ASEE 2023."
+        />
+
+        <ResearchCard
+          title="Investigating the effects of testing frequency on programming performance and students' behavior"
+          referenceHref="https://doi.org/10.1145/3545945.3569821"
+          reference="D. Smith, C. Emeka, M. Fowler, M. West, and C. Zilles. ASEE 2023."
+        />
+
+        <ResearchCard
+          title="First try, no (autograder) warm up: motivating quality coding submissions"
+          referenceHref="https://peer.asee.org/43714"
+          reference="L. Butler and G. Herman. ASEE 2023."
+        />
+      </React.Fragment>
+    ),
+  },
+
+  {
+    title: "Computer-based testing facilities",
```

Comment:
Are we going to standardize on the use of "center" rather than "facility", like we do for the marketing page?
```suggestion
    title: "Computer-based testing centers",
```